crates/primitives/src/attr.rs

```diff
@@ -1,5 +1,59 @@
-use bitflags::bitflags;
+//! Constraints and Column attributes.
+//!
+//! ## For tables
+//!
+//! The database engine support a sub-set of constraints defined by the `SQL` standard:
+//!
+//! ### UNIQUE:
+//!
+//!  One or many columns with enforced uniqueness using an auto-generate index.
+//!
+//! ### IDENTITY:
+//!
+//! A column that is made up of values generated by the database using a sequence.
+//!
+//!  Differs from `PRIMARY_KEY` in that its values are managed by the database and usually cannot be modified.
+//!
+//! ### PRIMARY_KEY:
+//!
+//! One or many columns that uniquely identifies each record in a table.
+//!
+//! Enforce uniqueness using an auto-generate index.
+//!
+//! Can only be one per-table.
+//!
+//! ## For Columns
+//!
+//! Additionally, is possible to add markers to columns as:
+//!
+//! - AUTO_INC: Auto-generate a sequence
+//! - INDEXED: Auto-generate a non-unique index
```

Comment:
Note: `AUTO_INC` right now is semantically ambiguous.  it clashes with the purpose of `IDENTITY` (and is possible to have many of it) which is the recommended one when asked for auto-generated numbers. 

I think we should use instead `SEQUENCE` and deprecate the use of this tag.